### PR TITLE
Upgrade to RSpec 3.3

### PR DESCRIPTION
**Pull request by alext**
Apart from keeping up to date, this will make it easier to upgrade to Rails 4.2

### CODE REVIEW

File: spec/features/finding_support_options_spec.rb [hunk 1]
```diff
@@ -139,18 +137,18 @@
       select "Education", :from => "sectors"
       select "Grow and sustain", :from => "stages"
 
-      page.should have_selector('.filter-results-summary h3 span', text: '1')
-      page.find('li.scheme h3').text.should == "Graduate start-up"
+      expect(page).to have_selector('.filter-results-summary h3 span', text: '1')
+      expect(page.find('li.scheme h3').text).to eq("Graduate start-up")
```
Comment (boffbowsh) [inline spec/features/finding_support_options_spec.rb]:
```ruby
expect(page).to have_selector('li.scheme h3', text: "Graduate start-up")
```
would be better here I think

File: spec/features/finding_support_options_spec.rb [hunk 2]
```diff
@@ -139,18 +137,18 @@
...
-      page.should have_selector('.filter-results-summary h3 span', text: '0')
+      expect(page).to have_selector('.filter-results-summary h3 span', text: '0')
 
       select "0 - 9", :from => "business_sizes"
       select "Manufacturing", :from => "sectors"
 
-      page.should have_selector('.filter-results-summary h3 span', text: '1')
-      page.find('li.scheme h3').text.should == "Manufacturing Services - Wales"
+      expect(page).to have_selector('.filter-results-summary h3 span', text: '1')
+      expect(page.find('li.scheme h3').text).to eq("Manufacturing Services - Wales")
```
Comment (boffbowsh) [inline spec/features/finding_support_options_spec.rb]: Same as above

**Comment by boffbowsh** [comment]
Tiny pedantic comments, otherwise :+1:

File: spec/features/finding_support_options_spec.rb (hunk 1 above)
Comment (alext) [inline spec/features/finding_support_options_spec.rb]: This is just what transpec did, but you're right. I'll update.

Comment (alext) [inline spec/features/finding_support_options_spec.rb]: Updated and rebased.

**Comment by boffbowsh** [comment]
:+1: but no merge access

**Comment by jackscotti** [comment]
merging

**Pull request merged**